src/components/ActionButton.tsx

```diff
@@ -11,6 +11,11 @@ const StyledButton = styled(Button)`
   flex-grow: 1;
   max-height: 56px;
   transition: background-color 0.25s ease-out, border-radius 0.25s ease-out, flex-grow 0.25s ease-out;
+  ${({ theme, disabled }) =>
```

Comment:
you're already writing css. Could you simplify this, or do you need to wrap it to avoid overwriting the background-color. eg

    background-color: ${({ disabled, theme }) => disabled && theme.interactive };

Reply:
Yes, was avoiding an overwrite here